doc/tutorial/using_gpu.txt

```diff
@@ -470,7 +133,7 @@ the GPU object directly.  The following code is modifed to do just that.
 
   rng = numpy.random.RandomState(22)
   x = shared(numpy.asarray(rng.rand(vlen), config.floatX))
-  f = function([], gpuarray.basic_ops.GpuFromHost(None)(tensor.exp(x)))
+  f = function([], tensor.exp(x).transfer('dev0'))
```

Comment:
Should that be `.transfer(None)`? That is what the text below says, and I don't think `dev0` was introduced in any way.